### PR TITLE
Update metrics and readiness docs

### PR DESCRIPTION
**Pull request by mfadul24**
### Summary by CodeRabbit

- New Feature: Introduced a "readiness check" endpoint that provides real-time status of worker availability. It returns an HTTP 200 status code if there is at least one worker ready to take a request, and an HTTP 500 status code if all workers are busy or unavailable. The default not-ready status code can be customized.
- Documentation: Added detailed explanation about the internal request queue mechanism in RR (presumably a web server). This includes how `allocate_timeout` is used to assign workers to requests and the process for rejecting requests when the timeout is exceeded.

### CODE REVIEW

File: docs/en/http/http.md [hunk 1]
```diff
@@ -525,3 +525,9 @@ http:
 
 So in this case the request gets into the `sendfile` middleware, then `gzip` and `static`. And vice versa from the
 response.
+
+## Request queues
+
+RR has an internal queue for requests. The `allocate_timeout` is used to assign a worker to the request. 
+If your worker works for 1 minute for example, but allocate_timeout is equal to 30seconds, after this timeout, RR will start rejecting the first request in queue.
```
Comment (rustatian) [inline docs/en/http/http.md]:
```suggestion
If your worker works for 1 minute for example, but `allocate_timeout` is equal to 30 seconds, after this timeout, RR will start rejecting the first request in queue. 
```

**Comment by mfadul24** [comment]
@rustatian I've updated the PR. Look forward to my first contribution :) 
Have a great day.

**Review by rustatian — APPROVED** [review]
LGTM

**Comment by rustatian** [comment]
Thanks @mfadul24 👍🏻

**Pull request merged**